setup.py

```diff
@@ -4,58 +4,37 @@
-if 'USE_PYODBC' in os.environ and os.environ['USE_PYODBC'] == '1':
-    require = ['sqlalchemy>=0.7.3']
-else:
-    require = ['sqlalchemy>=0.7.3','ibm_db>=2.0.0']
-    
+readme = os.path.join(os.path.dirname(__file__), 'README.md')
 
 setup(
-         name='ibm_db_sa',
-         version=VERSION,
-         license='Apache License 2.0',
-         description='SQLAlchemy support for IBM Data Servers',
-         author='IBM Application Development Team',
-         author_email='opendev@us.ibm.com',
-         url='http://pypi.python.org/pypi/ibm_db_sa/',
-         download_url='http://code.google.com/p/ibm-db/downloads/list',
-         keywords='sqlalchemy database interface IBM Data Servers DB2 Informix IDS',
-         long_description_content_type='text/markdown',
-         classifiers=[
-            'Development Status :: 4 - Beta',
-            'Intended Audience :: Developers',
-            'License :: OSI Approved :: Apache License 2.0',
-            'Operating System :: OS Independent',
-            'Topic :: Databases :: Front-end, middle-tier'
-        ],
-         long_description=open(readme).read(),
-         platforms='All',
-         install_requires= require,
-         packages=['ibm_db_sa'],
-        entry_points={
-         'sqlalchemy.dialects': [
-                     'db2=ibm_db_sa.ibm_db:DB2Dialect_ibm_db',
-                     'db2.ibm_db=ibm_db_sa.ibm_db:DB2Dialect_ibm_db',
-                     'db2.zxjdbc=ibm_db_sa.zxjdbc:DB2Dialect_zxjdbc',
-                     'db2.pyodbc=ibm_db_sa.pyodbc:DB2Dialect_pyodbc',
-                     'db2.zxjdbc400=ibm_db_sa.zxjdbc:AS400Dialect_zxjdbc',
-                     'db2.pyodbc400=ibm_db_sa.pyodbc:AS400Dialect_pyodbc',
-
-                     # older "ibm_db_sa://" style for backwards
-                     # compatibility
-                     'ibm_db_sa=ibm_db_sa.ibm_db:DB2Dialect_ibm_db',
-                     'ibm_db_sa.zxjdbc=ibm_db_sa.zxjdbc:DB2Dialect_zxjdbc',
-                     'ibm_db_sa.pyodbc=ibm_db_sa.pyodbc:DB2Dialect_pyodbc',
-                     'ibm_db_sa.zxjdbc400=ibm_db_sa.zxjdbc:AS400Dialect_zxjdbc',
-                     'ibm_db_sa.pyodbc400=ibm_db_sa.pyodbc:AS400Dialect_pyodbc',
-                    ]
-       },
-       zip_safe=False,
-       tests_require=['nose >= 0.11'],
-     )
+    name='sqlalchemy-ibmi',
+    version=VERSION,
+    license='Apache License 2.0',
+    description='SQLAlchemy support for Db2 on IBM i',
+    author='IBM',
+    keywords='sqlalchemy database ibm ibmi db2',
+    long_description_content_type='text/markdown',
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache License 2.0',
+        'Operating System :: OS Independent',
+        'Topic :: Databases :: Front-end, middle-tier'
+    ],
+    long_description=open(readme).read(),
+    platforms='All',
+    install_requires=['sqlalchemy>=1.3',
+                      'pyodbc>=4.0'],
```

Comment:
I'd prefer to see this formatted like classifiers or entry_points, where the entries are on separate lines from their brackets

Like
```python
   install_requires=[
      'sqlalchemy>=1.3',
      'pyodbc>=4.0',
],
```